src/commands/check.ts

```diff
@@ -0,0 +1,45 @@
+import { colours } from "../../deps.ts";
+import Module from "../interfaces/module.ts";
+import { constructModulesDataFromDeps } from "../utils.ts";
```

Comment:
instead of utils.ts, you could create a service class that does some logic for you pertaining to whatever it's named. like MolduleService. And then you could do:

```typescript
import { ModuleService } from "../services/module_service.ts;
ModuleService.constructModulesDataFromDeps( ... )
```

that way we get away from potentially storing a bunch of random functions in a utils class.

Reply:
I like that approach 👍 Will sort it out